tools/unsteady.go

```diff
@@ -55,6 +55,36 @@ func parseBCHeader(line string) (parentType string, parent string, flowEndRS str
 	return
 }
 
+// Return a RatingCurve object
+// Error handling?
+func getRatingCurveData(sc *bufio.Scanner, rc *RatingCurve) (error, bool) {
+	series, innerErr := getDataPairsfromTextBlock("Rating Curve", sc, 80, 8)
+
+	if innerErr != nil {
+		return innerErr, false
+	}
+	rc.Values = series
+
+	for sc.Scan() {
+		line := sc.Text()
+		loe := leftofEquals(line)
+
+		switch loe {
+		case "Boundary Location":
```

Comment:
Can you use the `forcingElementPrefix` to do this, because we will have more keywords that will signal the end of this element and start of new element other than `Boundary Location`